src/app/space/add-app-overlay/add-app-overlay.component.less

```diff
@@ -83,6 +83,21 @@ input,
         position: absolute;
         top: 96px;
       }
+      &_tooltip-label {
+        cursor: pointer;
+        float: right;
+        margin-top: -8px;
+        span {
+          margin-left: 5px;
+        }
+      }
+      .popover {
+        max-width: none;
+        width: 250px;
+        ul {
+          margin-left: -23px;
```

Comment:
Instead of using a negative margin, you can overwrite the default padding of the `<ul>` element by setting it to `padding: 18px;`. That will keep the desired alignment of the list, and use attributes that are already part of the list element.